CMakeLists.txt

```
@@ -16,6 +16,18 @@ add_compile_definitions(USE_YAML=TRUE)
 
 find_library(FFTW3 fftw3 REQUIRED)
 
+# find astshim library using EUPS package environent variable as hint
+find_library(ASTSHIM_LIBRARIES
+astshim REQUIRED
+HINTS $ENV{ASTSHIM_DIR}/lib
+)
+#find astshim include directory
+find_path(
+ASTSHIM_INCLUDES
+astshim.h REQUIRED
+HINTS $ENV{ASTSHIM_DIR}/include
+)
+
```

I can't comment on this line below
This adds the target `WCSfit` as an executable
linking all static libs. `astrometry` which links `astshim`
Linking everything into one shlib `gbdes` might work and link this to this executable. Or remove
this target if it is not needed.

```suggestion
 # add_subdirectory(src)
```

---

I don't think we need an executable target here, unless @cmsaunders still finds it convenient for comparison-testing sometimes.

---

I'm going to just leave in instructions for how to build the executable if needed.